src/SignalR.Orleans/Core/ServerDirectoryGrain.cs

```diff
@@ -67,7 +69,7 @@ private async Task ValidateAndCleanUp(object serverDirectory)
 		var expiredServers = State.Servers.Where(server => server.Value < DateTime.UtcNow.AddMinutes(-Constants.SERVERDIRECTORY_CLEANUP_IN_MINUTES)).ToList();
 		foreach (var server in expiredServers)
 		{
-			var serverDisconnectedStream = _streamProvider.GetStream<Guid>(server.Key, Constants.SERVER_DISCONNECTED);
+			var serverDisconnectedStream = _streamProvider.GetStream<Guid>(Constants.SERVER_DISCONNECTED, server.Key);
```

Comment:
seems inconsistent with the other, need to see api GetStream now if they changed order also (as im seeing you switched orders also)

Reply:
should be fixed - API is first namespace then the id